src/renderers/shared/fiber/ReactFiberScheduler.js

```diff
@@ -355,7 +385,15 @@ module.exports = function<T, P, I, TI, C>(config : HostConfig<T, P, I, TI, C>) {
       ReactFiberInstrumentation.debugTool.onWillBeginWork(workInProgress);
     }
     // See if beginning this work spawns more work.
-    let next = beginWork(current, workInProgress, nextPriorityLevel);
+    let next;
+    const isFailedWork = capturedErrors && capturedErrors.has(workInProgress);
```

Comment:
I thought we made the decision that recovering from error work should use whatever priority created the error in the first place?